Daytime-Nighttime-Interactions/slack-channel-usage.rst

```diff
@@ -0,0 +1,192 @@
+
+For items having substantial impact, scheduling should first be carried out as part of the `SUMMIT Jira calendar <https://jira.lsstcorp.org/secure/DoItBetterCalendar.jspa>`_ process.
+
+**Announcement Protocol:**
+
+- Announcements are required for all work that may affect the summit control network or its infrastructure.
+  This includes hardware testing, CSC updates/deployments, configuration changes, Nublado updates, invasive networking changes and any on-site activities being performed by summit personnel.
+- Utilize threads whenever possible to minimize cross-talk and/or interference between simultaneous discussions
+
+**Announcement Guidelines and Procedure:**
+
+- Make the announcement when ready to begin work.
+  Include an estimate of how long the systems will be in use.
+  Include the names of other channels where conversation on the work will take place (if appropriate).
+  List the CSC(s) you plan on using and a 1 or 2-sentence synopsis of what you are doing.
+
+- Announce when work is completed.
+  This should not be buried in a previous thread but instead should be a new message.
+  
+- Announcements for testing using other test-stands are to be made on test-stand specific channels, following the same rules.
+  This includes #rubinobs-tucson-teststand and #rubinobs-base-teststand
+
+
+#summit-control-room
+--------------------
+This room is actively monitored by observers and shift managers.
+
+Communications in this room should be minimal.
+This is a single channel for both telescopes which is actively monitored by telescope specialists and anyone controlling the system.
+For maximum efficiency, primary communication should be via Zoom and not in the channel.
+Breakout rooms can be used if multiple simultaneous conversations are required.
+
+This channel is meant to be used as the area where the copying/pasting of code snippets happens while supporting via zoom, therefore context will often not be provided.
+Troubleshooting in this area should be either system-wide, or issues which benefit from having multiple people see and understand the diagnosis.
+
+An example of troubleshooting belonging to this channel is found in the following figure.
+
+.. figure:: ./_static/dialog_example.png
+    :name: #summit-control-room example
+    :width: 560
+    :height: 399
+
+    An example of the type of troubleshooting conversation that should take place in #summit-control-room.
+
+
+#summit-maintel and #summit-auxtel
+----------------------------------
+
+These rooms are for telescope system specific asynchronous conversation with remote support.
+They are passively monitored by on-shift observers with no expectation of an immediate response.
+Generally, the content will be read and start and end of a shift, or during quiet periods.
```

Comment:
Not sure I understood this phrase. Did you mean:

```
Generally, the content will be read at the start and end of a shift, or during quiet periods.
```

e.g.: "... will be read and start..." -> "... will be read at start..."